examples/pico_w_explorer/pico_w_explorer_vector.cpp

```diff
@@ -41,8 +41,8 @@ int main() {
 
     vector.draw(poly);
 
-    //pp_mat3_t t = pp_mat3_identity();
-    //vector.text("Hello World", {0, 0}, &t);
+    pp_mat3_t t = pp_mat3_identity();
+    vector.text("Hello World", 0, 0, &t);
```

Comment:
Possibly uncommenting this was unwise as I don't have one of these to test.  I got confused between this and the new RP2350 based explorer 😆